### PR TITLE
[Fix #174] make clean-ns comptabile with project clean

### LINKED ISSUE #174: when using clean-ns with project-clean some ` *temp file* files get created
reported by @martinrojer. i was able to reproduce. these temp files are created for in two subdirectories of my project. content seems to be coming from two namespaces of my project (none of the is really changed by the clean)

### PR DESCRIPTION
**Pull request by benedekfazekas**
not entirely happy with this but could not come up with a better solution. any advice @expez @magnars ?

### CODE REVIEW

**Comment by expez** [comment]
What problem is this solving?

File: clj-refactor.el [hunk 1]
```diff
@@ -2228,10 +2228,14 @@ root."
   (interactive)
   (cljr--assert-leiningen-project)
   (cljr--assert-middleware)
-  (save-buffer)
-  (let ((result (nrepl-send-sync-request
+  (when (not (boundp 'filename))
+    (save-buffer))
+  (let* ((path-to-file (if (boundp 'filename)
```
Comment (expez) [inline clj-refactor.el]: I believe this is constantly false.  I can't find a variable named `filename` when I check.

**Comment by benedekfazekas** [comment]
solves #174 

filename is defined in a macro called `cljr--update-file` used by `cljr-project-clean`

**Comment by benedekfazekas** [comment]
to reproduce the problems just configure you project-clean to run clean-ns and run it

**Pull request merged**